test/integration/redirects_test.rb

```diff
@@ -77,4 +77,20 @@ def test_destroy_article_put
     assert_equal(article_url(Article.first.id),
                  @response.header["Location"])
   end
+
+  def test_show_glossary_term
```

Comment:
I tested just a sample of the old actions.
Should I test them all? 
If I do that for all controllers, it could noticeably slow the test suite, as these are all integration tests.

Reply:
More tests are better.  My test suite at work takes over 40 minutes to run and we do not have as good coverage as MO.  Having a test suite that runs in under 5 minutes is a goal for some folks, but realistically that's a very hard bar to meet and you essentially end up creating a subset of tests that you use for local runs and then you let the CI/CD system run the whole suite.  I'd much rather go that way than worry about reducing the speed of the test suite.

Reply:
I will test redirects of all the legacy actions.